app/components/page_content_component.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="page-content-message">
+    <p><i class="fa-solid fa-asterisk" style="color: #FEC51D;"></i> <%= page_content.value %> <%= link_to page_content.link_text, page_content.link_url if page_content.link_visible %></p>
```

Comment:
Can we not put the `style` in here?

Reply:
like make the color a css class and then add it here instead of inlining via style tag?

Reply:
Yes, use a CSS class rather than an inline style.

Reply:
ok, changed to css style